e2e/tests/ui-driven/src/create-flow/helpers.ts

```diff
@@ -0,0 +1,39 @@
+import { Browser, Page, Request } from "@playwright/test";
+import { createAuthenticatedSession } from "../globalHelpers";
+
+export const getUserRequest = (req: Request) => {
+  const isHasuraRequest = req.url().includes("/graphql");
+  const isGetUserRequest = Boolean(
+    isHasuraRequest && req.postData()?.toString().includes("GetUserById"),
+  );
+  return isGetUserRequest;
+};
```

Comment:
_Question / Naming:_

This function is `getUserRequest` but it seems like it takes in a request as an argument and checks whether it's a user request? Rather than literally getting the request? If this is the case would it make more sense to reflect that in the name?

Reply:
Good catch thanks - renamed!